services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -81,10 +85,9 @@ async def docker_compose_config(
     [SEE docker-compose](https://docs.docker.com/engine/reference/commandline/compose_convert/)
     [SEE compose-file](https://docs.docker.com/compose/compose-file/)
     """
-    # NOTE: TIMEOUT adjusted because of https://github.com/docker/compose/issues/3927, https://github.com/AzuraCast/AzuraCast/issues/3258
     result: CommandResult = await _write_file_and_spawn_process(
```

Comment:
please rename that method, since AFAIU your PR intention, is to not use a file right? so are you using a file or are you not using a file?
Also I think those commands to run shell also take ENV arguments, you might want to use that for your timeout options...